WooCommerce/Classes/ViewRelated/ReusableViews/TextFieldContentAlignment.swift

```diff
@@ -0,0 +1,22 @@
+import UIKit
+
+/// Horizontal alignment for content (text or placeholder) in a text field.
+enum TextFieldContentAlignment {
+    case leading
+    case trailing
+    case center
+}
+
+extension TextFieldContentAlignment {
+    func toTextAlignment(layoutDirection: UIUserInterfaceLayoutDirection = UIApplication.shared.userInterfaceLayoutDirection) -> NSTextAlignment {
```

Comment:
Were you planning to add a test for this? Is that why you exposed the `layoutDirection` argument?

Reply:
yes I did plan to add tests for this, but thought this mapping feels straightforward. it's easy to add tests though, so I added them in https://github.com/woocommerce/woocommerce-ios/pull/2442/commits/1e378d324ae0088c28309c6471d46615905e6988